### PR TITLE
Fix project formatting

### PR DESCRIPTION
**Pull request by Lessthanfree**
Attempt to fix overall project formatting so that our code can pass CI

### CODE REVIEW

**Comment by Lessthanfree** [comment]
Related to #138

File: src/main/java/seedu/address/commons/core/AddressbookComponentManager.java [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 /**
  * Base class for *Manager classes
- *
+ * <p>
```
Comment (yujiatay) [inline src/main/java/seedu/address/commons/core/AddressbookComponentManager.java]: What is the use of `<p>` here?

File: src/main/java/seedu/address/commons/core/ComponentManager.java [hunk 2]
```diff
@@ -4,7 +4,7 @@
 
 /**
  * Base class for *Manager classes
- *
+ * <p>
```
Comment (yujiatay) [inline src/main/java/seedu/address/commons/core/ComponentManager.java]: Same as above

File: src/main/java/seedu/address/commons/core/index/Index.java [hunk 3]
```diff
@@ -2,7 +2,7 @@
 
 /**
  * Represents a zero-based or one-based index.
- *
+ * <p>
```
Comment (yujiatay) [inline src/main/java/seedu/address/commons/core/index/Index.java]: Same p tag

File: src/main/java/seedu/address/commons/exceptions/IllegalValueException.java [hunk 4]
```diff
@@ -13,7 +13,7 @@ public IllegalValueException(String message) {
 
     /**
      * @param message should contain relevant information on the failed constraint(s)
-     * @param cause of the main exception
+     * @param cause   of the main exception
```
Comment (yujiatay) [inline src/main/java/seedu/address/commons/exceptions/IllegalValueException.java]: extra space

File: src/main/java/seedu/address/commons/util/JsonUtil.java [hunk 5]
```diff
@@ -51,7 +51,8 @@
     /**
      * Returns the Json object from the given file or {@code Optional.empty()} object if the file is not found.
      * If any values are missing from the file, default values will be used, as long as the file is a valid json file.
-     * @param filePath cannot be null.
+     *
+     * @param filePath                   cannot be null.
```
Comment (yujiatay) [inline src/main/java/seedu/address/commons/util/JsonUtil.java]: Is it a requirement to have the extra spacing?

File: src/main/java/seedu/address/ui/HelpWindow.java [hunk 6]
```diff
@@ -41,21 +41,21 @@ public HelpWindow() {
...
+     *                               <li>
+     *                               if this method is called during animation or layout processing.
+     *                               </li>
+     *                               <li>
+     *                               if this method is called on the primary stage.
+     *                               </li>
+     *                               <li>
+     *                               if {@code dialogStage} is already showing.
+     *                               </li>
+     *                               </ul>
```
Comment (yujiatay) [inline src/main/java/seedu/address/ui/HelpWindow.java]: This looks less visually pleasing. Travis has weird taste?

File: src/test/data/XmlAnakinStorageTest/invalidAndValidDeckAnakin.xml [hunk 7]
```diff
@@ -10,7 +10,7 @@
     </decks>
     <!-- Deck with invalid name-->
     <decks>
-        <name> </name>
+        <name></name>
```
Comment (yujiatay) [inline src/test/data/XmlAnakinStorageTest/invalidAndValidDeckAnakin.xml]: I think this is part of the test

File: src/test/data/XmlAnakinStorageTest/invalidDeckAnakin.xml [hunk 8]
```diff
@@ -2,7 +2,7 @@
 <anakin>
     <!--Deck with invalid name field-->
     <decks>
-        <name> </name>
+        <name></name>
```
Comment (yujiatay) [inline src/test/data/XmlAnakinStorageTest/invalidDeckAnakin.xml]: Same as above

File: src/test/data/XmlSerializableAnakinTest/invalidDeckAnakin.xml [hunk 9]
```diff
@@ -2,7 +2,7 @@
 <anakin>
     <!--Deck with invalid name field-->
     <decks>
-        <name> </name>
+        <name></name>
```
Comment (yujiatay) [inline src/test/data/XmlSerializableAnakinTest/invalidDeckAnakin.xml]: Please check for the rest

Comment (Lessthanfree) [reply src/test/data/XmlSerializableAnakinTest/invalidDeckAnakin.xml]: Noted and changed

File: src/main/java/seedu/address/commons/core/index/Index.java (hunk 3 above)
Comment (Lessthanfree) [reply src/main/java/seedu/address/commons/core/index/Index.java]: Weird. I think its a side-effect of the IntelliJ autoformat

File: src/main/java/seedu/address/commons/exceptions/IllegalValueException.java (hunk 4 above)
Comment (Lessthanfree) [reply src/main/java/seedu/address/commons/exceptions/IllegalValueException.java]: Part of the auto-formatter to make descriptions flushed. So "of" is aligned with "should"

**Pull request merged**